hax/unblinding.py

```diff
@@ -57,6 +57,9 @@
 unblind['dec'] = {}
 unblind['dec']['e_range'] = '((0.0137*(cs1/.1429 + cs2_bottom/11.36) < 50.) | (0.0137*(cs1/.1429 + cs2_bottom/11.36) > 80.))'
 
+# Unblinding of 14 days post-AmBe data for I-125 removal assessment
```

Comment:
Please add this PR number (i.e. "see #215") to the comment for future reference.